index.js

```diff
@@ -41,6 +41,7 @@ const Admin = require('./models/Admin');
 const Team = require('./models/Team');
 const Story = require('./models/Story');
 const Event = require('./models/Event');
+const Idea = require('./models/Idea');
 
 app.use('/', require('./routes/index'));
 app.use('/auth', require('./routes/auth'));
```

Comment:
You have to import the route here and use that route to map with the function.

Reply:
The route file routes/api/v1/web_api is already imported in index.js and /api/v1/ideas is described in web_api

Reply:
okay, that works